modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryServiceTest.java

```diff
@@ -849,17 +853,34 @@ private void _assertUserNotificationEventsCount() throws PortalException {
 		long[] userIds = _userLocalService.getRoleUserIds(role.getRoleId());
 
 		for (long userId : userIds) {
-			int count =
-				_userNotificationLocalService.getUserNotificationEventsCount(
+			List<UserNotificationEvent> userNotificationEvents =
+				_userNotificationLocalService.getUserNotificationEvents(
 					userId, portletId,
 					LocalDate.now(
 					).atStartOfDay(
 						ZoneId.systemDefault()
 					).toInstant(
 					).getEpochSecond(),
-					true, "%\"exceedsObjectEntryLimit\":%");
+					true);
 
-			Assert.assertTrue(count == 1);
+			int count = 0;
+
+			for (UserNotificationEvent userNotificationEvent :
+					userNotificationEvents) {
+
+				if (count > 0) {
```

Comment:
Hey @thallescezar, correct me if I'm wrong, but if you keep this condition with the continue statement, the count variable will never reach any number greater than 1, so you're not testing the case/possible bug when the admin user gets 2 or more notifications.